File: dataset_reduzido/avatar/Python/Code/codeforces_171_A.py
import sys
input = sys.stdin.readline
a , b = input ( ) [ : - 1 ].split ( )
x = max ( len ( a ) , len ( b ) )
a = a.rjust ( x , '0' )
b = b.ljust ( x , '0' )
s = ''
c = 0
for i in range ( x ) :
    d = int ( b [ i ] ) + int ( a [ x - i - 1 ] ) + c
    if d > 9 :
        s += str ( d ) [ 1 ]
        c = 1
    else :
        s += str ( d )
        c = 0
if c == 1 :
    s += '1'
print ( int ( s [ : : - 1 ] ) )

File: output/LLaMa/avatar/Python/Java/codeforces_171_A.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split("");
        int x = Math.max(input[0].length(), input[1].length());
        String a = input[0];
        String b = input[1];
        while (a.length() < x) {
            a = "0" + a;
        }
        while (b.length() < x) {
            b = "0" + b;
        }
        StringBuilder s = new StringBuilder();
        int c = 0;
        for (int i = x - 1; i >= 0; i--) {
            int d = (int) b.charAt(i) + (int) a.charAt(x - i - 1) + c;
            if (d > 9) {
                s.insert(0, d % 10);
                c = 1;
            } else {
                s.insert(0, d);
                c = 0;
            }
        }
        if (c == 1) {
            s.insert(0, '1');
        }
        System.out.println(s.reverse().toString());
    }
}